CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Helpers/ButtonConfiguration.swift

```diff
@@ -74,10 +74,10 @@ public extension UIButton {
         self.layer.borderWidth = configuration.borderWidth
         self.layer.shadowRadius = configuration.shadowRadius
 
+        // When switching from one ButtonConfiguration with a blur effect to another ButtonConfiguration with a blur effect, // the previous blur effect should be removed.
```

Comment:
[request] Please move this `// the previous blur effect should be removed.` on the second line